### ISSUE TITLE
Explore using ViewPagerAndroid instead of ScrollView/FlatList

### ISSUE THREAD

**Issue by zachgibson** [zachgibson/react-native-parallax-swiper#12]
Not sure if this is viable, but the scrolling animation is *way* better on Android using [ViewPagerAndroid](https://facebook.github.io/react-native/docs/viewpagerandroid.html) versus ScrollView. Current issues are:

1. ViewPagerAndroid exposes scroll offset but gets reset to 0 on every page
2. ViewPagerAndroid probably isn’t optimized for long lists like [FlatList](https://facebook.github.io/react-native/docs/flatlist.html)
    - Could possibly still add this as an option if wanted e.g. short list of like 3 pages

**Comment by YashGadle** [comment]
I tried your component to display images from my gallery. So I rendered around 100 images but it started to lag. Maybe use Animated Flatlist instead of Animated.ScrollView?

**Comment by zachgibson** [comment]
Yeah, this is definitely the plan moving forward. I won’t be able to get to it immediately but PRs are definitely welcome. 🙌
